swarm/src/behaviour.rs

```diff
@@ -79,16 +79,11 @@ pub(crate) type THandlerOutEvent<THandler> =
 /// it will delegate to each `struct` member and return a concatenated array of all addresses
 /// returned by the struct members.
 ///
-/// When creating a custom [`NetworkBehaviour`], you must choose one of two methods to respond to
```

Comment:
Note that the documentation on `NetworkBehaviour` now no longer documents `NetworkBehaviourEventProcess`. My rational being that we will deprecate it anyways, thus there is no use in pointing newcomers to it.

Any objections?

Reply:
Nope.